prog/postgres/postgres_nexus.rb

```diff
@@ -134,6 +134,16 @@ def before_run
     nap 5
   end
 
+  label def update_superuser_password
+    escaped_password = postgres_server.superuser_password.gsub(/['\"]/, "'" => "''", "\"" => "\\\"")
```

Comment:
Any suggestions to handle this more gracefully?

Reply:
does it need to? SQL is not Unix.
```
fdr=# select '"';
 ?column? 
----------
 "
(1 row)
```

Reply:
It does, because in next line, I use it in `vm.sshable.cmd()`. Here is a scary example;

```bash
clover-development(main)> password = "\"; echo whoami; #"
clover-development(main)> vm.sshable.cmd("sudo -u postgres psql -c \"ALTER USER postgres WITH PASSWORD #{DB.literal(password)}\"")

ERROR:  unterminated quoted string at or near "'"
LINE 1: ALTER USER postgres WITH PASSWORD '
                                          ^
ubi
=> "ubi\n"
```

Reply:
Well... DB.literal does not, as it is specialized for SQL, but I need something that does :smile:

Reply:
You need shellescape. There's two different quoting systems layered here.

Reply:
I would probably cook up the SQL separately, and then `shellescape` the whole thing. I wonder if we should like into an argv variant of `#cmd` that could accept `%w[sudo -u postgres psql -c] + ["ALTER USER...."]`. That would require digging around in net-ssh's feature set just a bit, and some type switches if available.